custom_tensorflow_keras_nlp/Headline Classifier SageMaker.ipynb

```diff
@@ -489,10 +490,10 @@
     "]\n",
     "\n",
     "estimator = TensorFlowEstimator(\n",
-    "    framework_version=\"1.15\",\n",
-    "    py_version=\"py3\",\n",
+    "    framework_version=\"2.4\",\n",
+    "    py_version=\"py37\",\n",
     "    instance_count=1,\n",
-    "    instance_type=\"ml.g4dn.xlarge\",\n",
+    "    instance_type=\"ml.m5.xlarge\",\n",
```

Comment:
I know there's been some discussion about the trade-off between likelihood of hitting quota/capacity errors with GPUs and large audiences, versus wanting to show accelerated computing in the workshop: But I checked with the other ASEAN owners and our position for now is still to stick with GPU by default.

Perhaps we could add a comment directly after the line to make switching simpler for users? E.g.
```python
estimator = TensorFlowEstimator(
    instance_type="ml.g4dn.xlarge",  # Or CPU-only "ml.c5.xlarge", if you see quota/capacity errors
)
```

Could go for m5.xlarge instead of c5, but I didn't think the extra memory was needed in these exercises?